src/ConfigGen/Program.cs

```diff
@@ -470,7 +470,7 @@ static string createConsumerSpecific()
         ///     <see cref=""Confluent.Kafka.Consumer{TKey,TValue}.Consume(System.TimeSpan)"" />
         ///     method. Disabling fields that you do not require will improve
         ///     throughput and reduce memory consumption. Allowed values:
-        ///     headers, timestamp, topic, all, none
+        ///     headers, timestamp, topic, status, all, none

```

Comment:
The message status field is currently only relevant to producer delivery reports.
A consumed message is per definition persisted.

Reply:
Yeah :+1: